rocAL/rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -2563,11 +2564,31 @@ rocalAudioFileSourceSingleShard(
                                                                                         shard_size
                                                                                         );
         context->master_graph->set_loop(loop);
+        if(downmix) {
+            TensorInfo output_info = info;
+            std::vector<size_t> output_dims = {context->user_batch_size(), info.dims()[1], 1};
+            // std::vector<size_t> output_dims;
+            // output_dims.resize(3);
+            // output_dims.at(0) = context->user_batch_size();
+            // output_dims.at(1) = info.dims()[1];
+            // output_dims.at(2) = 1;
+            output_info.set_dims(output_dims);
+            output_info.set_tensor_layout(RocalTensorlayout::NONE);
+            output_info.set_max_shape();
```

Comment:
In set_dims if set_max_shape call is present, we need not call again here check that